static/css/style.css

```diff
@@ -156,4 +156,7 @@ footer ul li a {
 .page-item.active .page-link {
     background-color: var(--p-orange);
     border-color: var(--p-orange);
-}
\ No newline at end of file
+}
+
+/* Landing Page */
```

Comment:
Was there any custom CSS styles added to the landing page, or was this comment optional just in case? If no custom styles, I don't think we need a blank comment here then.